src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
Use * selector for box-sizing, usually you need to change logic of all elements sizing